enterprise-ws-spec/src/main/asciidoc/Handlers.adoc

```diff
@@ -101,11 +88,9 @@ document either in its business logic or in message handlers.
 
 === Specification
 
-This section defines the requirements for JAX-RPC or JAX-WS Handlers
-running in Web Services for Java EE. Chapter 12 of the JAX-RPC
-specification and Chapter 9 of the JAX-WS specification defines the
-programming model requirements. Differences between this specification
-and the JAX-RPC or JAX-WS specification are noted in boxed paragraphs.
+This section defines the requirements for Jakarta XML Web Services Handlers 
+running in Web Services for Jakarta EE. Differences between this specification 
```

Comment:
"...running in Jakarta Enterprise Web Services." instead? Given that "Another name for the Web Services for Java EE specification is the Java Specification Requirements (JSR) 109." - quoted [from here](https://www.ibm.com/support/knowledgecenter/SSEQTP_8.5.5/com.ibm.websphere.base.doc/ae/cwbs_j2ee.html)

Reply:
yes, agree, will change it to Jakarta Enterprise Web Services.